gix/tests/repository/config/transport_options.rs

```diff
@@ -106,6 +107,9 @@ mod http {
                 max: version
             })
         );
+
+        assert!(ssl_verify);
```

Comment:
Please use the second argument to `assert!()` for a message that explains that the default is `true` here - this is the value of this assertion really, as an accidental 'false if unset' would be a terrible security issue.